sys/arlib

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
```

Comment:
oh well. there are some bashisms out here :( its ok to go for bash then. this file has been ripped from cccl right? will be good to rewrite using posix shell